TestHelpers.Tests/MockFileSetAttributesTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace System.IO.Abstractions.TestingHelpers.Tests
+{
+    using XFS = MockUnixSupport;
```

Comment:
I'd prefer if this was moved up with the other usings, for consistency.

Reply:
I prefer the same. I only put it here for consistency with other test files:

https://github.com/System-IO-Abstractions/System.IO.Abstractions/blob/0caa9382a68061a62c69ed3ef8f6f8204204ea7e/TestHelpers.Tests/MockDirectoryTests.cs#L8